create_sources/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, CMake
+
+
+class HelloConan(ConanFile):
+    name = "hello"
+    version = "0.1"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake"
+    scm =  {
+        "type": "svn",
+        "url": "auto",
+        "revision": "auto",
+    }
```

Comment:
This will copy the whole contents of the repository as sources. Although the create command is working with the build and package commands using the "create_sources" subdirectories, we may want to have a proper git repo only to show the proper usage of this feature. We can discuss about this in our next meeting